Core/include/Acts/Seeding/SpacePointGrid.hpp

```diff
@@ -46,6 +46,8 @@ struct SpacePointGridConfig {
   // return 1 neighbor on either side of the current phi-bin (and you want to
   // cover the full phi-range of minPT), leave this at 1.
   int phiBinDeflectionCoverage = 1;
+  // maximum number of phi bins
+  int maxPhiBins = std::numeric_limits<int>::max();
```

Comment:
this is a lot. Maybe we can put a more reasonable default? like 200?

Reply:
I can set it to 200 but then it will become a breaking change, right?

Reply:
why? it's just setting a default value

Reply:
Because it could cause performance changes to clients?